lib/util/package-rules/api.ts

```diff
@@ -0,0 +1,38 @@
+import { BaseBranchesMatcher } from './base-branches';
+import { CurrentVersionMatcher } from './current-version';
+import { DatasourcesMatcher } from './datasources';
+import { DepTypesMatcher } from './dep-types';
+import { FilesMatcher } from './files';
+import { LanguagesMatcher } from './languages';
+import { ManagersMatcher } from './managers';
+import { PackageNameMatcher } from './package-names';
+import { PackagePatternsMatcher } from './package-patterns';
+import { PackagePrefixesMatcher } from './package-prefixes';
+import { PathsMatcher } from './paths';
+import { SourceUrlPrefixesMatcher } from './sourceurl-prefixes';
+import { SourceUrlsMatcher } from './sourceurls';
+import type { MatcherApi } from './types';
+import { UpdateTypesMatcher } from './update-types';
+
+const api = new Map<string, MatcherApi[]>();
```

Comment:
i don't see any usage for the keys. shouldn't we use a simple array here?
```suggestion
const api MatcherApi[][] = [];
```